build_installers.py

```diff
@@ -254,6 +254,9 @@ def _napari_env(
             f'napari-menu={napari_version}',
             'napari-plugin-manager',
             f'pyside2={pyside_version}',
+            # Support extra file formats
+            'imagecodecs',
+            'ffmpeg',
```

Comment:
Should these be added to the `napari` output in the recipe (not in `napari-base`)?